TurkishFilter/sections/antiadblock.txt

```diff
@@ -1,6 +1,8 @@
 !
 ! These rules are against all kinds of ad blocking reinjection/circumvention scripts
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/75398
+kanald.com.tr#%#//scriptlet("set-constant", "hasAdblock", "false")
```

Comment:
Please move it to the existing rule and change number of issue:
Like this
```
! https://github.com/AdguardTeam/AdguardFilters/issues/75398
kanald.com.tr#%#//scriptlet("set-constant", "hasAdblock", "false")
@@||kanald.com.tr/*scripts/ad*.js
```

Reply:
Ah yes thanks!